### ISSUE TITLE
photo migration into PROD and issue with photos in Dev

### ISSUE THREAD

**Issue by CrystalChadburn** [bcgov/invasivesbc#2548]
- [x] Dev only has some photos showing.  It appears it may be only the photos attached to the site record, but we require all photos from each site to show in InvasivesBC including those that were attached to the survey or treatment records in IAPP.  Are they stored differently?
- [ ] There are currently zero photos in PROD.  Need to migrate all photos after the dev issues is solved.

This is required before launch.

**Comment by CrystalChadburn** [comment]
@plasticviking   Hi Rob,  During testing this issue was found.  If you have any questions or need support on figuring this out please let myself, mike or brennan know.  This is a key requirement for launch!

**Comment by plasticviking** [comment]
@CrystalChadburn No issues discovered yet. I am working on the queries to include the missing photos today.

**Comment by CrystalChadburn** [comment]
@plasticviking Thank you so much!

**Comment by plasticviking** [comment]
@CrystalChadburn PR 2581 completes the linkages and will be visible in dev after it is approved and merged. Of the 21,707 images imported, 21,256 now link to site records (451 are "orphaned" with no obvious reference to a valid site or treatment id). This is up from 11,258/21,707 linked before this PR.

Once this is approved and merged and if no further changes are required, I will do the prod import.

**Comment by CrystalChadburn** [comment]
Thanks @plasticviking!   We have confirmed that Brennan needs to re-make the extracts to fix some of the treatment and survey ID numbers which will fix the remaining issues with photos attached to wrong records and the orphan issues.  No further action for you until we get the IAPP details fixed, and then hopefully that will automatically fix the issues we are having and then we can push to prod.

**Comment by plasticviking** [comment]
Thanks for confirming @CrystalChadburn . I'll park this for now and work on other issues.

**Comment by CrystalChadburn** [comment]
made a new ticket for the PROD migration
